openshift/kustomize/tekton/base/tasks/deploy-all.yaml

```diff
@@ -45,7 +45,7 @@ spec:
       image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
       resources: {}
       script: |
-        set -xe
+        # set -xe
```

Comment:
is this meant to be commented out?

Reply:
Yes, it's for debugging.  It adds a lot of noise.